pkg/adapter/adapter.go

```diff
@@ -128,6 +126,25 @@ func (l listener) handleEvent() http.HandlerFunc {
 		var logger *zap.SugaredLogger
 
 		l.event = info.NewEvent()
+
+		// if repository auto configuration is enabled then check if its a valid event
+		if l.run.Info.Pac.AutoConfigureNewRepo {
+			detected, configuring, err := github.ConfigureRepository(ctx, l.run, request, string(payload), l.logger)
+			if detected {
+				if configuring && err == nil {
+					l.writeResponse(response, http.StatusCreated, "configured")
+					return
+				}
+				if configuring && err != nil {
+					l.logger.Errorf("failed to configured repository, err: %v", err)
+					l.writeResponse(response, http.StatusOK, "failed to configure")
+					return
+				}
+				l.writeResponse(response, http.StatusOK, "skipped event")
```

Comment:
this would look weird in log can you write which event has been skipped?

Reply:
we don't log this here,  this is api response
we log here https://github.com/openshift-pipelines/pipelines-as-code/blob/08fecbdc074050fb085a1f0551dfc7888bafc0c6/pkg/provider/github/repository.go#L41
for skipped repo events